content/app/how-tos/ingress-with-whitelist.md

```diff
@@ -0,0 +1,76 @@
++++
```

Comment:
I'm not familiar with the platform but few questions:
1. is source IP address not NATed? will the platform users always see external IP? if it is coming form the pod I assume they'll but what about external traffic?
2. what is the internalTrafficPolicy set on the cluster? if it is set to cluster your source IP migh be SNATed, so the IP you put in that list might differ. 
Feel free to ignore all those if this was already tested/discussed and designed into the model, just not aware of the design.